src/Weasel.Postgresql/Tables/TableColumn.cs

```diff
@@ -181,12 +181,19 @@ public string FullDeclaration()
     }
 }
 
-public class SerialValue: ColumnCheck
+public class SerialValue : ColumnCheck
 {
-    public override string Declaration()
-    {
-        return "SERIAL";
-    }
+    public override string Declaration() => "SERIAL";
+}
+
+public class BigSerialValue : ColumnCheck
+{
+    public override string Declaration() => "BIGSERIAL";
+}
+
+public class SmallSerialValue: ColumnCheck
+{
+    public override string Declaration() => "SMALLSERIAL";
 }
```

Comment:
Are these types actually needed?
If I try to remove them the build and tests pass.

If they're not needed should I remove them?

Reply:
Good catch. They're not used in the project, but someone outside of Weasel could use them. Let's keep them for now, I'll revisit them while working on the next major version.